### ISSUE TITLE
Wrong package name user-management

### ISSUE THREAD

**Issue by afranzi** [thoughtspot/cs_tools#36]
### First Stop

- [X] I have read and agreed to the ThoughtSpot Dev Tools EULA.
- [X] I understand that if I have a critical issue with ThoughtSpot, I should reach out to my account team instead of cs_tools.
- [X] I searched the cs_tools documentation, with the integrated search.

### Platform Configuration

---

### Example Command

---

### Description

The [user-management](https://github.com/thoughtspot/cs_tools/blob/master/cs_tools/cli/tools/user-management) package has a `-` instead of a `_` so it cannot be imported properly.

Could be possible to follow PEP8 conventions on the package names?

Thank you! :)

**Comment by boonhapus** [comment]
@afranzi this is an intentional design decision -- the tools are meant to be standalone, mini programs, essentially. Our target audience for this library aren't developers, but ThoughtSpot admins / consumers of the data itself. Many design decisions have been made with this kept in mind. I've considered adding [ADRs][design] to this repository to help communicate some of those decisions.

I'm curious again, what you're hoping to achieve? 😄 I'd like to help, but want to guide you appropriately for the situation.

[design]: https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions.html

**Comment by afranzi** [comment]
We are automatizing the user & group provisioning since some of our teams have some kind of `dynamic` nature. So we would like to remove any manual work.

**Comment by boonhapus** [comment]
What's the general process you'd envision for automating user/group provisioning?

Depending on what your process looks like, you might be better off interacting directly with the [ThoughtSpot User/sync API][user-sync] itself.

We have [an interface to the tools][i-cs-tools] themselves. I feel this will not be simple to implement though, since the tools are meant to be interacted with on the command line. It might be better offer simulating interaction with them using subprocess and parameterizing that...

Something like this should help if you want to write code directly against the tools. 

```python
from cs_tools.const import TOOLS_DIR
from cs_tools.cli.loader import CSTool


def main():
    user_mgmt = CSTools(TOOLS_DIR / "user-management")
    # do stuff with the tool..
   

if __name__ == "__main__":
    raise SystemExit(main())
```

[i-cs-tools]: https://github.com/thoughtspot/cs_tools/blob/4d4ce2b7279fd9f83047469a48310e44c8bd5041/cs_tools/cli/loader.py#L12
[user-sync]: https://developers.thoughtspot.com/docs/?pageid=user-api#user-sync

**Comment by boonhapus** [comment]
This is resolved as part of V1.4.0 . Check the release notes to learn more.

https://thoughtspot.github.io/cs_tools/changelog/1.4.0
